Fix lintian warning:
lldpad: hyphen-used-as-minus-sign usr/share/man/man8/lldptool-app.8.gz:18--- a/docs/lldptool-app.8
+++ b/docs/lldptool-app.8
@@ -15,7 +15,7 @@
 .TP
 .B app
 Configures a local application data field using the three parameters
-priority (prio), selector (sel), and protocol (pid). The -d option
+priority (prio), selector (sel), and protocol (pid). The \-d option
 can be provided to remove an entry which matches the three parameters.
 
 .SH Theory of Operations
